tests/test_main.py

```diff
@@ -161,7 +161,7 @@ def test_arthur_and_lancelot_have_potion(base_knights_config):
     base_knights_config["arthur"]["potion"] = {
         "name": "Dragon's heart",
         "effect": {
-            "protection": +20,
+            "add_some_abilities": +20,
```

Comment:
Ensure that the key 'add_some_abilities' is correctly used and consistent with the rest of the code. If this is a typo or inconsistency, it may lead to unexpected behavior.